apps/main/src/llamalend/features/market-list/cells/PriceCell.tsx

```diff
@@ -72,6 +72,13 @@ export const PriceCell = ({ getValue, row, column }: CellContext<LlamaMarket, nu
                 (stats?.borrowToken?.amount ?? 0) * (stats?.borrowToken?.usdRate ?? 0),
               loading: isLoading,
             }}
+            collateralLoss={{
+              depositedCollateral: stats?.collateralLoss?.depositedCollateral,
+              currentCollateralEstimation: stats?.collateralLoss?.currentCollateralEstimation,
+              percentage: stats?.collateralLoss?.percentage,
+              amount: stats?.collateralLoss?.amount,
+              loading: isLoading,
+            }}
```

Comment:
Less boiler plate == less chance for errors
```suggestion
            collateralLoss={{ ...stats?.collateralLoss, loading: isLoading }}
```

Reply:
yes you can ser
<img width="161" height="60" alt="image" src="https://github.com/user-attachments/assets/240692d1-33cc-4457-9cd0-fa06f457f9f7" />

Reply:
This I can do

<img width="233" height="120" alt="SCR-20251126-sqmo" src="https://github.com/user-attachments/assets/00748b4c-24bb-4c57-a5cb-1d356f38a393" />